### DISCUSSION TITLE
Ability to clear log file from admin GUI

### DISCUSSION THREAD

**Discussion by ralmond** [nextcloud/all-in-one#5659] [Ideas] [answered]
I'm using Nextcloud Hub 8 (29.0.7).  My log file is showing 9929 errors, so opening the log file in my browser brings my browser to its knees.

I'm not seeing way to clear/truncate/rotate the log file from the admin GUI, and I don't have command line access to the server, so the the solution reported here https://github.com/nextcloud/all-in-one/discussions/4654 doesn't work for me.

So I'm resubmitting this as an RFE.

**Comment by szaimen** [comment, marked as answer]
Hi, please create a feature request for this here: https://github.com/nextcloud/logreader/issues. Thanks!
